clusters/demo/base/kustomization.yaml

```diff
@@ -6,9 +6,9 @@ resources:
   - ../../../apps/toffee/base/kustomize.yaml
   - ../../../apps/pip/base/kustomize.yaml
   - ../../../apps/kube-system/base/kustomize.yaml
-  - ../../../apps/my-time/base/kustomize.yaml
+  # - ../../../apps/my-time/base/kustomize.yaml
```

Comment:
why dont we remove the line?

Reply:
we giong to put it back once the upgrade is done, at the moment it has failing pods which does not allow us to do the upgrade, I will revert this pr in next 15-20 minutes